packages/backend/src/database/entities/savedSemanticViewerCharts.ts

```diff
@@ -71,9 +71,9 @@ export type DBSavedSemanticViewerChartVersion = {
     saved_semantic_viewer_chart_version_uuid: string;
     saved_semantic_viewer_chart_uuid: string;
     created_at: Date;
-    config: object;
+    config: Record<string, unknown>;
```

Comment:
inferring objects or {} when generating swagger causes warnings. I just followed instructions to get rid of all warnings

Reply:
should a type from backend entities be exposed on the API ? we normally don't expose these. 

And also, I remember tsoa was not able to handle Record types, it was simply removing their values, which was hard to debug.